app/views/users/show.html.erb

```diff
@@ -1 +1,39 @@
-<h1>All users show</h1>
+<!-- app/views/users/show.html.erb -->
+<h1>User Details</h1>
+
+<div class="row justify-content-around">
+  <div class="card width-40">
+    <%= image_tag(@user.photo, alt: 'User Photo') if @user.photo.present? %>
+  </div>
+  <div class="flex-row width-60 align-items-baseline gap-2 card-padding-10">
+    <div class="card card-padding-10">
+      <h2><%= @user.name %></h2>
+      <p class="d-flex justify-content-end ">Number of posts <%= @user.posts.count %></p>
+    </div>
+  </div>
+</div>
```

Comment:
👍 Excellent work on the functionality! Now, let's focus on refining the UI to match the wireframe more closely. The user details card is a recurring element in three out of the four wireframes, and ensuring consistency in its design is important. Here are some suggestions:

- [ ] [Optional] **Consistent User Details Card:** Compare the wireframe with the actual implementation, and please make sure that the user details card is consistent across the different views. Please pay attention to details such as font size, spacing, and layout to closely match the wireframe design.
- [ ] [Optional] **Use Rails Partials:** Additionally, consider using [Rails partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) to break the rendering process into more manageable chunks. With a partial, you can move the code for rendering a particular piece of a response to its file. This can help to keep your code DRY (Don't Repeat Yourself) and make it more maintainable For example as the user card is the same in different places, you can create a partial for that and use it in different places.

|Pages | Your | Design |
| --- | --- | --- |
|  Users| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/61a0f0cf-dea6-4fbe-84b3-600c0ebcbab0)| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/f538dbd7-9d29-41e0-9c8f-6263f841b3a8)|
|  User Details| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/8962c263-39a3-4ce9-a1a6-fdf5eeb5e188)| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/a18f253e-59a9-4b5b-97d2-bbda31cccca4)|
|  User Posts| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/2f64051e-141f-4e2d-bfac-850a2a28e709)| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/8bb5bfc9-3ef3-4c76-96d9-d18bd7453220)|

By aligning the UI more closely with the wireframe and leveraging Rails partials, you'll enhance the visual coherence of your application and improve code organization. Keep up the great work! 🚀🎨